packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set modalDefault.twig__1.html

```diff
@@ -3,7 +3,10 @@
   <div class="Modal__content">
     <div class="Modal__dialog">
       <div class="Modal__header">
-        <button aria-controls="modal-example" aria-expanded="false" data-dismiss="modal" data-target="#modal-example" class="Button Button--tertiary  Button--square" type="button">          <span class="Modal__close" aria-hidden="true"></span>
+        <button aria-controls="modal-example" aria-expanded="false" data-dismiss="modal" data-target="#modal-example" class="Button Button--tertiary  Button--square" type="button">          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewbox="0 0 24 24" fill="none" id="f4c60a9bc97c06a47dd828bfc4909ab5" class="Icon ">
```

Comment:
Button has `inline-flex` then as `Button--square` has `flex: none` - my question is: The redundant spaces inside the button before and after svg tag should be trimmed, since the svg is not `display: block` or am I wrong?

Reply:
Maybe it would be better, but as the Square button has a set width it doesn't matter afaik. We can fix it in a separate issue.